example/src/App.css

```
@@ -1,225 +1,708 @@
+
+.runner-panel select,
+.runner-panel textarea {
+  border-color: #426159;
+  color: #eef8f5;
+  background: #193a32;
+}
+
+textarea {
+  min-height: 190px;
+  resize: vertical;
+  font-family: "SFMono-Regular", Consolas, monospace;
```

Stylelint (17.14.1)

[error] Expected no quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)